core/commands/configure_command.rb

```diff
@@ -66,6 +75,15 @@ def configure_rhel
     SUCCESS_RESULT
   end
 
+  def input_docker_credentials
+    {
+      'username' => read_topic('Please input username for Docker Registry'),
```

Comment:
Please provide previously inputted username and password as default values.

Update code for other products to do this as well.